utils/validate-facility-ids/database-client/mongo-db-client.ts

```diff
@@ -0,0 +1,21 @@
+import z from 'zod';
+import { MongoDbClient } from '@ukef/dtfs2-common/mongo-db-client';
+
+const mongoDbConfigSchema = z.object({
+  MONGODB_URI_QA: z.string(),
+  MONGO_INITDB_DATABASE: z.string(),
+});
+
+/**
+ * Gets the mongo db client
+ * @returns The mongo db client
+ */
+const getMongoDbClient = (): MongoDbClient => {
+  const mongoDbConfig = mongoDbConfigSchema.parse(process.env);
+  return new MongoDbClient({
+    dbName: mongoDbConfig.MONGO_INITDB_DATABASE,
+    dbConnectionString: mongoDbConfig.MONGODB_URI_QA,
+  });
+};
+
+export const mongoDbClient = getMongoDbClient();
```

Comment:
Noticed there is already a DB client in `utils/mock-data-loader/database` - maybe could extra this into the `utils` directory (or the other way around) to be DRY?

Reply:
Each of the mongo clients may technically need to use different connection strings depending on the task being performed, hence each util having its own db client. If we want to commonise this further down the line then we could, but the files are so small that there isn't much harm in keeping the multiple versions for now